chore(docs): add gateway collector brief overview (#41)
liatrio/tag-o11y-quick-start-manifests: README.md

To deploy the basic set of configuration with the LGTM stack and a Gateway Open Telemetry collector using **Tilt** - a Microservice Deployment Engine - run `make`.

## Gateway Collector

> The gateway collector is created using an  Open Telemetry Collector fork that Liatrio maintains called the Liatrio Otel Collector. The gateway collector is configured to receive, process, and export the 3 keys of Observability, metrics, logs and traces.

Reviewer: ```suggestion
The gateway collector is created using an  OpenTelemetry Collector fork that Liatrio maintains called the Liatrio OTel Collector. The gateway collector is configured to receive, process, and export the three observability signals;  metrics, logs and traces.
```

> It is able to...

Reviewer: ```suggestion
In the default quick start stack, the gateway collector:
```

> Receive metrics and process/export them to Prometheus.

Reviewer: Same change should be made to the other two below, bulleted list, no new lines between.

Receive logs and process/export them to Loki. 

Receive traces and process/export them to Tempo, Jaeger.

> Why do you want to use the Gateway collector? This collector is the entry point to making use of this observability stack. Sending your telemetry data here will route it to where it needs to go in order for you to see and use it. For example, sending metrics to the gateway collector will store them in the Prometheus Database, allowing you to build Grafana dashboards that can query those metrics and visualize them. 

Reviewer: ```suggestion
Why do you want to use the [Gateway][gw] collector? This collector is the entry point to forwarding telemetry to the analysis backends.

[gh]: https://opentelemetry.io/docs/collector/deployment/gateway/
```

To deploy the basic set of configuration with the LGTM stack and a Gateway Open Telemetry collector using **Tilt** - a Microservice Deployment Engine - run `make`.

## Gateway Collector
The gateway collector is created using an  OpenTelemetry Collector distribution that Liatrio maintains called the Liatrio OTel Collector. The gateway collector is configured to receive, process, and export the three observability signals;  metrics, logs and traces.

In the default quick start stack, the gateway collector:

* receives metrics and processes/exports them to Prometheus.

> 

Reviewer: ```suggestion
```

* receives logs and processes/exports them to Loki. 

> 

Reviewer: ```suggestion
```
